test/Libraries/RevitNodesTests/Elements/GroupTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Autodesk.DesignScript.Geometry;
+using NUnit.Framework;
+using Revit.Elements;
+using RevitTestServices;
+using RTF.Framework;
+
+namespace RevitNodesTests.Elements
+{
+    [TestFixture]
+    public class GroupTests : RevitNodeTestBase
+    {
+        private const double Tolerance = 0.001;
+
+        [Test]
+        [TestModel(@".\element.rvt")]
+        public void CanCreateGroupFromListOfElements()
+        {
+            // Arrange
+            List<int> elementIds = new List<int>() { 184176, 184324 };
+            List<Element> elementsToGroup = elementIds.Select(id => ElementSelector.ByElementId(id, true)).ToList();
+
+            // Act
+            var group = Group.FromElements(elementsToGroup);
+
+            // Assert
+            Assert.AreEqual(207894, group.Id);
```

Comment:
we should also add check that uses Revit API to `GetMembers` and `CollectionAssert.AreEqual(elementIds, memberIds)`